source/codegen/metadata/nifake/functions_addon.py

```diff
@@ -4,4 +4,20 @@
 functions_override_metadata = {
 }
 
-
+functions_validation_suppressions = {
+    "AcceptViSessionArray": {
+        "parameters": {
+            "sessionArray": ["INPUT_ARRAY_SHOULD_NOT_HAVE_PASSED_IN_SIZE"]
```

Comment:
note: These are just in `nifake` so I could have "fixed" them, but it's nice to have some tests that make sure we can still generate code for cases like this.